Detectors/TPC/workflow/src/TPCTimeSeriesSpec.cxx

```diff
@@ -187,6 +190,15 @@ class TPCTimeSeries : public Task
     if (mBufferDCA.mOccupancyMapTPC.size() > mMaxOccupancyHistBins) {
       mBufferDCA.mOccupancyMapTPC.resize(mMaxOccupancyHistBins);
     }
+    /// TRD tracklets
+    const auto & tracksTRD = mTPCOnly ? gsl::span<o2::trd::TrackTRD>() : recoData.getITSTPCTRDTracks<o2::trd::TrackTRD>();
+    const auto& trklets = mTPCOnly ? gsl::span<const o2::trd::Tracklet64>():recoData.getTRDTracklets();
+    std::unordered_map<unsigned int, unsigned int> indicesTRDTPC; // TPC track index -> TRD-TPC track index.
+    // loop over all TRD tracks and map them to TPC tracks
+    for (int i = 0; i < tracksTRD.size(); ++i) {
+      //auto trackTPC = tracksTRD[i].getRefTPC().getIndex();
+      //if (trackTPC >= 0) indicesTRDTPC[trackTPC] = i;
+    }
```

Comment:
What exactly do you want to store? TRD track are using not the raw (`o2::trd::Tracklet64`) tracklets but calibrated ones, see https://github.com/AliceO2Group/AliceO2/blob/dev/Detectors/TPC/calibration/SpacePoints/src/TrackInterpolation.cxx#L445-L452 for the access to tracklets associated to the track.